backend/swpp/tests.py

```diff
@@ -38,3 +40,43 @@ def test_user_profile_tutor_relation(self):
         self.assertTrue(hasattr(user.profile, 'tutor'))
         self.assertTrue(hasattr(user.profile.tutor, 'profile'))
         self.assertEqual(user.profile, user.profile.tutor.profile)
+
+    # added 05/01, from modelInit branch
+
+    def test_valid_profile(self):
+        user = self.create_user('s', '2')
+        data = {'major': '', 'contact': '010-1111-1111'}
+        serializer = ProfileSerializer(user.profile, data = data, partial = True)
+        self.assertFalse(serializer.is_valid())
+
+        data['major'] = 'bio'
+        serializer = ProfileSerializer(user.profile, data = data, partial = True)
```

Comment:
매번 새로 Serializer를 파는거보단 serializer 내부 데이터를 직접 바꾸는 것이 편리합니다.
ex)
```suggestion
        serializer.object.major = 'bio'
```

Reply:
serializer.object가 아마 없고 .data로 있었던거 같지만 다음 pull request 전에 반영하도록 하겠습니다

Reply:
serializer.validated_data['major'] = 'bio'
였던 것 같기도 하고...

Reply:
이 경우엔 아직 validate 안 된거니까 그냥 data일수도

Reply:
그냥 원래대로 하는게 제일 나은 것 같네요 실험해보니
죄송합니다